src/Protocol/PortableContact.php

```diff
@@ -1174,17 +1174,19 @@ public static function checkServer($server_url, $network = "", $force = false)
 				if (!empty($data->channels_total)) {
 					$registered_users = $data->channels_total;
 				}
-				switch ($data->register_policy) {
-					case "REGISTER_OPEN":
-						$register_policy = REGISTER_OPEN;
-						break;
-					case "REGISTER_APPROVE":
-						$register_policy = REGISTER_APPROVE;
-						break;
-					case "REGISTER_CLOSED":
-					default:
-						$register_policy = REGISTER_CLOSED;
-						break;
+				if (!empty($data->register_policy)) {
+					switch ($data->register_policy) {
+						case "REGISTER_OPEN":
+							$register_policy = REGISTER_OPEN;
+							break;
+						case "REGISTER_APPROVE":
+							$register_policy = REGISTER_APPROVE;
+							break;
+						case "REGISTER_CLOSED":
+						default:
+							$register_policy = REGISTER_CLOSED;
```

Comment:
Is `$register_policy` set before this? If `$data->register_policy` isn't set than the variable wouldn't either according to this.

Reply:
It is defined before.